src/tsickle.ts

```diff
@@ -359,13 +388,15 @@ class Annotator extends ClosureRewriter {
     let annotated = this.getOutput();
 
     let externsSource: string|null = null;
-    if (externs.output) {
+
+    if (externs.output.length > 0 || this.exportingDecoratorExterns.length > 0) {
       externsSource = `/**
  * @externs
  * @suppress {duplicate}
  */
 // NOTE: generated by tsickle, do not edit.
-` + externs.output;
+` + externs.output +
```

Comment:
I tested this piece of code, and it seems to do exactly what I mean it to do.

Adding `@export` (manually) on the fields seems to achieve the same result, but I have no clue whether things are handled even remotely similarly.